### PR TITLE
Combine DC Power examples into one that has a .measui and a .vi

### PR DESCRIPTION
**Pull request by dixonjoel**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Combine the two DC Power examples. They were the same except for the UI files. Use the .measui file as the default and explain how to switch to the LV UI.

### Why should this Pull Request be merged?

Minimize duplication on the two examples.

### What testing has been done?

Loaded the service in InstrumentStudio with both kinds of UIs

### CODE REVIEW

**Review by DelpireNI — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,8 +59,7 @@ The `examples` directory contains the following example projects:
 
 - `sample_measurement`: Performs a loopback measurement with various data types.
 - `nidaqmx_analog_input`: Performs a finite analog input measurement with NI-DAQmx.
-- `nidcpower_source_dc_voltage`: Sources and measures a DC voltage with an NI SMU. Provides a Measurement UI
-- `nidcpower_source_dc_voltage_with_labview_ui`: Sources and measures a DC voltage with an NI SMU. Provides a LabVIEW UI
+- `nidcpower_source_dc_voltage`: Sources and measures a DC voltage with an NI SMU. Provides a Measurement UI and a LabVIEW UI
```
Comment (bkeryan) [inline README.md:62]: Add a period

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: examples/nidcpower_source_dc_voltage/README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
   - Sources the same DC voltage level on all selected pin/site combinations
   - Measures the DC voltage and current for each selected pin/site combination
 - Includes InstrumentStudio and MeasurementLink UI Editor project files
+- Includes both a .measui (default) and LabVIEW .vi user interface. To change the
```
[the remark this answers is not in the document]
Comment (dixonjoel) [reply examples/nidcpower_source_dc_voltage/README.md:14]: Ok to keep this?
```
 To change the example to use the LabVIEW .vi, simply switch the order of the
  `ui_file_paths` array in measurement.py
```

[the remark this answers is not in the document]
Comment (bkeryan) [reply examples/nidcpower_source_dc_voltage/README.md]: Yes. Let's include the note about InstrumentStudio and the instruction for changing the example in both places.

File: examples/nidcpower_source_dc_voltage/README.md [hunk 3]
```diff
@@ -11,6 +11,9 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
   - Sources the same DC voltage level on all selected pin/site combinations
   - Measures the DC voltage and current for each selected pin/site combination
 - Includes InstrumentStudio and MeasurementLink UI Editor project files
+- Includes multiple UI files. Note: InstrumentStudio only displays the 1st UI file.
+  To change the example to use the LabVIEW .vi, simply switch the order of the
+  `ui_file_paths` array in measurement.py
```
Comment (bkeryan) [inline examples/nidcpower_source_dc_voltage/README.md:16]: style edits:
```suggestion
  To change the example to use the LabVIEW `.vi`, simply switch the order of the
  `ui_file_paths` array in `measurement.py`.
```

**Review by pbirkhol-ni — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -59,13 +59,12 @@ The `examples` directory contains the following example projects:
 
 - `sample_measurement`: Performs a loopback measurement with various data types.
```
Comment (bkeryan) [inline README.md:60]:
```suggestion
- `sample_measurement`: Performs a loopback measurement with various data types. Provides a Measurement UI and a LabVIEW UI.
```

**Review by bkeryan — APPROVED** [review]

**Pull request merged**